src/plone/api/tests/test_env.py

```diff
@@ -402,6 +402,12 @@ def test_test_mode(self):
         from plone.api.env import test_mode
         self.assertEqual(test_mode(), True)
 
+    def test_read_only_mode(self):
+        """Test that read_only_mode() returns False
```

Comment:
in fact I think you can just remove the docstring as it's pretty obvious what the test does.